scisim/static/js/Utils/PageChain.js

```diff
@@ -36,10 +41,29 @@ PageChain.prototype.revivePage = function(current, newPage){
 	this.chain[newPage].revive();
 };
 
+PageChain.prototype.revivePage = function(newPage){
```

Comment:
Javascript doesn't support method overloading, so you'll need to make two methods with two different names